pkgs/by-name/th/thrift/package.nix

```diff
@@ -111,31 +76,6 @@ stdenv.mkDerivation rec {
     "-DOPENSSL_USE_STATIC_LIBS=ON"
   ];
 
-  disabledTests = [
-    "PythonTestSSLSocket"
-    "PythonThriftTNonblockingServer"
-  ]
-  ++ lib.optionals stdenv.hostPlatform.isDarwin [
-    # Tests that hang up in the Darwin sandbox
-    "SecurityTest"
-    "SecurityFromBufferTest"
-    "python_test"
-
-    # fails on hydra, passes locally
-    "concurrency_test"
-
-    # Tests that fail in the Darwin sandbox when trying to use network
-    "UnitTests"
-    "TInterruptTest"
-    "TServerIntegrationTest"
-    "processor"
-    "TNonblockingServerTest"
-    "TNonblockingSSLServerTest"
-    "StressTest"
-    "StressTestConcurrent"
-    "StressTestNonBlocking"
-  ];
```

Comment:
Has this been tested with sandboxed Darwin? I’d be surprised if this doesn’t at least need `__darwinAllowLocalNetworking`. I’d also be hesitant about re‐enabling `concurrency_test` unless we expect it’ll perform better under load in the Hydra setting than it used to.

Reply:
This commit is from https://github.com/NixOS/nixpkgs/pull/374710

So yes @DaGenix said this was tested on sandboxed darwin. I did not double check that.